### PR TITLE
Add ping expression

### PR DESCRIPTION
**Pull request by Pikachu920**
Target Minecraft versions: Any
Requirements: Paper (or any other spigot fork with a getPing method under Player$Spigot)
Related issues: None

Description:
You can now get people's ping

### CODE REVIEW

**Comment by TheLimeGlass** [comment]
This doesn't exist in some versions between 1.7-1.10 for some unknown reason. Have you tested this on all versions because I can tell you there will be lots of errors because of no method found.

**Comment by Pikachu920** [comment]
can you please read the code? If the method doesn't exist it will error on init.

File: src/main/java/ch/njol/skript/expressions/ExprPing.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.Player;
+
+@Name("Ping")
+@Description("Returns a player's ping. This expression is only supported on some server software.")
+@Examples({"on join",
+			"\tbroadcast \"%player's ping\""})
```
Comment (Blueyescat) [inline src/main/java/ch/njol/skript/expressions/ExprPing.java]: missing %

**Comment by TheLimeGlass** [comment]
Still doesn't change the fact it's not very logical to add. Every user seeing this will want it added but in terms of the coding and probably future removal again by Spigot, it's not logical.

**Comment by Pikachu920** [comment]
I understand your concerns and even partially agree but the users who can't use this can install an addon that uses NMS for it. Also, if anything happens to this method it would be vanilla Spigot adding it, there's no reason that Paper would remove it as it's kinda one of the things they like to tout.

File: src/main/java/ch/njol/skript/expressions/ExprPing.java [hunk 2]
```diff
@@ -32,8 +32,9 @@
 
 @Name("Ping")
 @Description("Returns a player's ping. This expression is only supported on some server software.")
-@Examples({"on join",
-			"\tbroadcast \"%player's ping\""})
+@Examples({"command ping <player=%player%>:",
```
Comment (Blueyescat) [inline src/main/java/ch/njol/skript/expressions/ExprPing.java]: You forgot to make the argument optional, also `command /ping` :[ like the current sample scripts lol

Comment (Pikachu920) [reply src/main/java/ch/njol/skript/expressions/ExprPing.java]: fixed

File: src/main/java/ch/njol/skript/expressions/ExprPing.java (hunk 1 above)
Comment (Snow-Pyon) [reply src/main/java/ch/njol/skript/expressions/ExprPing.java]: was already merged so I'll fix it later

**Pull request merged**